tests/schema/extensions/test_extensions.py

```diff
@@ -541,3 +542,44 @@ def ping(self) -> str:
     assert result.errors == [GraphQLError("Well you asked for it")]
 
     assert mock_original_execute.call_count == 1
+
+
+def test_extend_error_format_example():
+    # Test that the example of how to extend error format
+
+    class ExtendErrorFormat(Extension):
+        def on_request_end(self):
+            result = self.execution_context.result
+            if hasattr(result, "errors") and result.errors is not None:
```

Comment:
This could be simplified to
```suggestion
            if getattr(result, "errors", None):
```
that's just a suggestion tho :)

Reply:
That's nice 😆